PublicMapping/districtbuilder-classic
# Reorder config to show desired fields in info popup

## Overview

C70 requested changes to the display of the info popup.  They would like to display the 5 following demographic data points:

Total Population
Total White Population
Total Black Population
Total Hispanic/Latino Population
Total Asian Population

This PR makes changes to `views.py` and `config.xml` to reorder the demographic Subjects and increase the number of elements listed on the info popup to include the ones they want.

### Checklist


### Notes

I made changes in what I believe to be the appropriate places to make this happen based on Kenny's message in #district-builder.  However I don't have a development environment to test these changes. I'm hoping this small change can save some developer time doing it, so I would like someone to test this out and see if the changes work. If it doesn't end up just working, I'll create a task in PivotalTracker for someone to take.

## Testing Instructions

 * With the app running on this branch, login and go to a plan
 * Click the info button and click a shape on the map
 * The 5 Subjects listed in the description above should be visible
